---
cve: 2017-3139
title: CVE in django-powerdns-dnssec
description: >
    A denial of service flaw was found in the way BIND handled DNSSEC validation. A remote attacker could use this flaw to make named exit unexpectedly with an assertion failure via a specially crafted DNS response.
cvss_v2: 5.0
references:
    - https://access.redhat.com/security/cve/cve-2017-3139
    - https://bugzilla.redhat.com/show_bug.cgi?id=1447743
affected:
    - name: django-powerdns-dnssec
      version:
        []
      fixedin:
        - ">=0.9.0"
